services/beeper-connector/beeper_to_rdf.py

```
@@ -0,0 +1,183 @@
+    except:
+        return user_id
+
+def get_thread_info(cursor, thread_id):
+    """Get thread information from the database."""
+    try:
+        cursor.execute("SELECT json_extract(thread, '$.title') FROM threads WHERE threadID = ?", (thread_id,))
+        result = cursor.fetchone()
+        if result and result[0]:
+            return result[0]
+        return thread_id
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)